app/api/services/suppliers.py

```diff
@@ -373,3 +382,56 @@ def get_approved_suppliers(self):
         )
 
         return [r._asdict() for r in results]
+
+    def get_business_info_by_abn(self, email_address, abn):
+        apiKey = current_app.config['ABR_API_KEY']
+        includeHistoricalDetails = 'N'
+        abn = abn
+        link = 'https://abr.business.gov.au/abrxmlsearch/AbrXmlSearch.asmx/SearchByABNv201205?searchString='
+        url = link + abn + '&includeHistoricalDetails=' + includeHistoricalDetails + '&authenticationGuid=' + apiKey
+
+        try:
+            response = requests.get(url)
+            response.raise_for_status()
+            xmlText = response.content
+            root = ElementTree.fromstring(xmlText)
+
+        # Rasing Different exceptions
+        except ConnectionError as ex:
+            raise AbrError('Connection Error')
+
+        # Invalid HTTP Reponse
+        except HTTPError as ex:
+            raise AbrError('HTTP Error')
+
+        except ProxyError as ex:
+            raise AbrError('ProxyError')
+
+        except Timeout as ex:
+            raise AbrError('Timeout')
+
+        except SSLError as ex:
+            raise AbrError('SSLError')
+
```

Comment:
What about the [exceptions returned in the payload](https://abr.business.gov.au/Documentation/Exceptions)?  How are we handling them?